micropython/net/webrepl/webrepl.py

```diff
@@ -110,11 +159,16 @@ def setup_conn(port, accept_handler):
 
 
 def accept_conn(listen_sock):
-    global client_s
+    global client_s, webrepl_ssl_context
     cl, remote_addr = listen_sock.accept()
+    req = cl.makefile("rwb", 0)
```

Comment:
Is this `makefile` necessary?  On bare-metal ports it just returns `cl`.  It's really only there for CPython compatibility, and I don't think this code here needs to be compatible with CPython.

Reply:
I've moved the code here from the old line 22, and yes it seems this is a no-op. I'll test this code with `makefile` removed.